apps/mark-scan/frontend/src/pages/remove_jammed_sheet_screen.tsx

```diff
@@ -2,15 +2,13 @@
 
 import { Caption, Icons, P } from '@votingworks/ui';
 import { CenteredCardPageLayout } from '../components/centered_card_page_layout';
-import { ResetVoterSessionButton } from '../components/deactivate_voter_session_button';
 
 export function RemoveJammedSheetScreen(): JSX.Element {
   return (
     <CenteredCardPageLayout
       icon={<Icons.Warning color="warning" />}
       title="Paper is Jammed"
       voterFacing={false}
-      buttons={<ResetVoterSessionButton />}
```

Comment:
Looks like there's a caption below that also needs to get moved or removed

>To end the current voter session and start over, press
     the button below to deactivate the voter session.

Reply:
Thanks for catching!